src/pudl/transform/classes.py

```diff
@@ -96,6 +96,7 @@ class Config:
         """Prevent parameters from changing part way through."""
 
         allow_mutation = False
+        extra = "forbid"
```

Comment:
Since `TableTransformParams` and `Ferc1TableTransformParams` both inherit from `TransformParams` I think the addition of this configuration here means we can remove it from those classes, where it's now redundant. Looks like it's only set in `Ferc1TableTransformParams` right now.

Reply:
oh good call